docs/operations/healthchecks.md

```diff
@@ -73,13 +73,19 @@ knows what's wrong.
 
 ### Endpoints
 
-We recommend using `/ready` and `/live` as the endpoints for the readiness and
+We recommend using `/readyz` and `/livez` as the endpoints for the readiness and
```

Comment:
```suggestion
We recommend that you use consistent naming for your endpoints across micro-services.  `/readyz` and `/livez` are common choices for the endpoints for the readiness and
```

Reply:
Discussed in Ref Arch team meeting today, we are doing a bit more research and will comment soon.